### PR TITLE
7656 GitHub ruleset issue template

### PR DESCRIPTION
**Pull request by JavierBejMen**
|Related issue|
|---|
|#7656|

## Description
Reworked templates for ruleset issues and PRs

- Removed old ruleset template and created new one only for rules/decoders requests.
- Added test checks for rules/decoders PRs

### CODE REVIEW

**Review by jcruzlp — DISMISSED** [review]
LGTM!

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+name: Ruleset issue
+about: Report a bug or make a feature request.
+title: ''
+labels: ''
+assignees: ''
+
```
Comment (NicoGuini) [inline .github/ISSUE_TEMPLATE/rules-decoders_request.md:7]: What about adding Github issue id?

**Review by 72nomada — DISMISSED** [review]
LGTM

**Review by fabamatic — DISMISSED** [review]
LGTM

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+---
```
Comment (NicoGuini) [inline .github/ISSUE_TEMPLATE/rules-decoders_request.md:8]: Are these lines needed?

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Contributions from the community are really appreciated. If this is the case, please add the
+"contribution" to properly track the issue.
+-->
+
+## Description
+<!-- Add a detailed description of your issue -->
+<!-- Detail the reason that motivates this proposed change on the ruleset -->
+
+### Service
+<!-- Add a description of the service you are targeting with the ruleset, indicating vendor and/or module if available -->
```
Comment (NicoGuini) [inline .github/ISSUE_TEMPLATE/rules-decoders_request.md:25]:
```suggestion
<!-- Add a description of the service that you are targeting with the ruleset, indicating vendor and/or module if available -->
```

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+This template reflects sections that must be included in new issues
+Contributions from the community are really appreciated. If this is the case, please add the
+"contribution" to properly track the issue.
+-->
+
+## Description
+<!-- Add a detailed description of your issue -->
+<!-- Detail the reason that motivates this proposed change on the ruleset -->
+
+### Service
```
Comment (NicoGuini) [inline .github/ISSUE_TEMPLATE/rules-decoders_request.md:24]: What about using Service/Product/Module

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+<!-- Add a description of the service you are targeting with the ruleset, indicating vendor and/or module if available -->
+<!-- Add any URL or doc related to service and/or vendor/module -->
+
+
+## Error/Improve
```
Comment (NicoGuini) [inline .github/ISSUE_TEMPLATE/rules-decoders_request.md:29]:
```suggestion
## Error/Improvements
```

**Review by NicoGuini — COMMENTED** [review]
Comments made

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md (hunk 2 above)
Comment (JavierBejMen) [reply .github/ISSUE_TEMPLATE/rules-decoders_request.md:8]:
```suggestion

```
Not really, it was inserted by error when copying the table below.

File: .github/ISSUE_TEMPLATE/rules-decoders_request.md (hunk 4 above)
Comment (JavierBejMen) [reply .github/ISSUE_TEMPLATE/rules-decoders_request.md:24]:
```suggestion
### Service/Product/Module
```
If it's clearer this way let's do it.

**Review by JavierBejMen — COMMENTED** [review]
Answered @NicoGuini comments

**Review by jcruzlp — APPROVED** [review]
LGTM!

**Review by 72nomada — APPROVED** [review]
LGTM

**Pull request merged**